assets/src/edit-story/components/panels/noSelection.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Panel, PanelContent } from './panel';
+
+const Note = styled.p`
+  font-style: italic;
+  color: ${({ theme }) => theme.colors.mg.v1};
```

Comment:
Figma file has a different color (#3C4043 not in the theme.js) and font-weight (300?) for this element.
Is there any other source of truth?

Reply:
It has `#3C4043` at `0.86` opacity on white background resulting in `#575b5d` with is not identical with but close to `mg.v1` at `#616877` but actually even closer to our `bg.v5` at `#575D65`.

So I'll change it to the latter.

At some future point we might need to give the entire theme an overhaul to align all colors with the still settling UX, but for now, that's close enough 😉

Reply:
Color and font weight fixed in b3c45980.

Reply:
Oh, right, I'm not used to `color`+`opacity` combo, I always just use `color` 😉 

Thing to remember, in Figma `font-weight: normal; === 300`